src/config.js

```diff
@@ -22,5 +27,5 @@ const config = {
 };
 
 // eslint-disable-next-line no-console
-console.info("config", config);
+console.info("config", JSON.stringify(config));
```

Comment:
Do we have no proper logger available?

Reply:
At the moment no. It is log it to the browser console. We will later wrap the logger into a util. But i do not know if we must replace it with any populated logger. It will pass additional kBit to the editor that do not more then the base console logger.
If any error is throw it is pass to the user information, or to sentry send us. But this stuff is not finished, only as prototype.